example/src/helpers/StrengthChecker.js

```diff
@@ -0,0 +1,52 @@
+const veryStrongPassword = new RegExp(
+  '(?=.*[A-Z].*[A-Z])(?=.*[!@#$&*])(?=.*[0-9].*[0-9])(?=.*[a-z].*[a-z].*[a-z]).{8}'
+);
+const strongPassword = new RegExp(
+  '(?=.*[A-Z].*[A-Z])(?=.*[!@#$&*])(?=.*[0-9].*[0-9])'
+);
+
+const goodPassword = new RegExp('(?=.*[A-Z].*[A-Z])(?=.*[!@#$&*])');
+
+const fairPassword = new RegExp('(?=.*[A-Z].*[A-Z])');
+
+export const StrengthChecker = (password) => {
```

Comment:
i think we should agree on a convention for function names
either UpperCamelCase (as u did it here) or lowerCamelCase

I vote for lowerCamelCase (strengthChecker)

Reply:
Also, the name should start with an action verb like checkPasswordStrength

Reply:
i put it on lowercase